state/postgres_server_stats.go

```diff
@@ -1,7 +1,42 @@
 package state
 
+import (
+	"database/sql"
+)
+
 // PostgresServerStats - Statistics for a Postgres server.
 type PostgresServerStats struct {
 	CurrentXactId   Xid8
 	NextMultiXactId Xid8
+
+	XminHorizonBackend                sql.NullInt32
+	XminHorizonReplicationSlot        sql.NullInt32
+	XminHorizonReplicationSlotCatalog sql.NullInt32
+	XminHorizonPreparedXact           sql.NullInt32
+	XminHorizonStandby                sql.NullInt32
+}
+
+// FullXminHorizonBackend - Returns XminHorizonBackend in 64-bit FullTransactionId
+func (ss PostgresServerStats) FullXminHorizonBackend() int64 {
+	return int64(XidToXid8(Xid(ss.XminHorizonBackend.Int32), Xid8(ss.CurrentXactId)))
```

Comment:
FWIW, it seems a bit odd to me that we're using a Nullable type above, but then we just ignore whether its flagged as `Valid` or not.

We're essentially relying on the zero value here being identical with the invalid transaction ID (0), but that may not be obvious to a future reader. If we can solve that on the Postgres side instead (with COALESCE, if we can make it work), that seems like a cleaner solution.